hedy.py

```diff
@@ -981,6 +981,28 @@ def NEGATIVE_NUMBER(self, args):
 
 
 class UsesPyGame(Transformer):
+    command_prefix = (f"""\
+pygame_end = False
+while not pygame_end:
+  pygame.display.update()
+  event = pygame.event.wait()
+  if event.type == pygame.QUIT:
+    pygame_end = True
+    pygame.quit()
+    break""")
+
+    regex_string_subsequent_use = (
+        r"\s*# End of PyGame Event Handler"
```

Comment:
This is a pretty smart move, for us to be able both if pressed consecutive commands and with other commands in between!